docs/interact/programmability/lua-api.md

```diff
@@ -344,17 +344,17 @@ It accepts a single input argument that only be one of the following:
 * `redis.REPL_NONE`: disables effect replication entirely.
 
 By default, the scripting engine is initialized to the `redis.REPL_ALL` setting when a script begins its execution.
-You can call the `redis.set_repl()` function at any time during the script's execution to switch between the different replication modes.
+You can call the `server.set_repl()` function at any time during the script's execution to switch between the different replication modes.
 
 A simple example follows:
 
 ```lua
-redis.replicate_commands() -- Enable effects replication in versions lower than Redis v7.0
-redis.call('SET', KEYS[1], ARGV[1])
-redis.set_repl(redis.REPL_NONE)
-redis.call('SET', KEYS[2], ARGV[2])
-redis.set_repl(redis.REPL_ALL)
-redis.call('SET', KEYS[3], ARGV[3])
+redis.replicate_commands() -- Enable effects replication in versions lower than Redis OSS v7.0
```

Comment:
Obviously I missed some Lua functions. I'll add another commit with those.

redis.replicate_commands
redis.debug
redis.acl_check_cmd

and some more